eng/cake/dotnet.cake

```diff
@@ -138,18 +138,6 @@ Task("dotnet-build")
         }
     });
 
-Task("dotnet-samples")
```

Comment:
let's keep this around

Reply:
I added the dotnet-samples cake task back, it will run all tests in the `SampleTests` class.

Reply:
Sorry I think we want to keep the build samples like it was to be used by other stuff. 

Can we add a task dotnet-samples-test , and call that and keep the botnet-samples to just build . 

Thanks